BridgeCareApp/VuejsApp/src/shared/utils/attributes.ts

```diff
@@ -0,0 +1,66 @@
+export const attributes: string[] = [
```

Comment:
These need to be populated from the database. Specifically the legacy database from the "ATTRIBUTES_" table.